includes/apple-exporter/class-parser.php

```diff
@@ -146,53 +146,62 @@ private function _parse_markdown( $html ) {
 	private function _clean_html( $html ) {
 		// Match all <a> tags via regex.
 		// We can't use DOMDocument here because some tags will be removed entirely.
-		preg_match_all( '/<a(?:.*?)(href|name)=(?:"|\')(.*?)(?:"|\')(?:.*?)>(.*?)<\/a>/m', $html, $a_tags );
+		preg_match_all( '/<a.*?>(.*?)<\/a>/m', $html, $a_tags );
 
-		// Check if we got matches
-		if ( empty( $a_tags ) ){
+		// Check if we got matches.
+		if ( empty( $a_tags ) ) {
 			return $html;
 		}
 
-		// Iterate over the matches and see what we need to do
-		foreach ( $a_tags[1] as $i => $tag_type ) {
-			// First, if this is an anchor, those aren't supported so just remove it.
-			if ( 'name' === $tag_type ) {
-				$html = str_replace( $a_tags[0][ $i ], $a_tags[3][ $i ], $html );
+		// Iterate over the matches and see what we need to do.
+		foreach ( $a_tags[0] as $i => $a_tag ) {
+			// If the <a> tag doesn't have content, dump it.
+			$content = trim( $a_tags[1][ $i ] );
```

Comment:
I may be reading this incorrectly, but I'm wondering if this should be `$a_tags[0][ $i ]` instead?

Reply:
`[0]` is going to contain the full match, but `[1]` will contain the content of the first capture group, which in this case is what is inside of the `a` tag. Personally, I think the PHP authors got the organization of the array backwards and that it's confusing, but so it goes :)